sources/packages/web/src/components/common/ScholasticStandingForm.vue

```diff
@@ -5,14 +5,17 @@
     @submitted="submitted"
   >
     <template #actions="{ submit }">
-      <footer-buttons
-        :processing="processing"
-        @primaryClick="submit"
-        primaryLabel="Submit update"
-        @secondaryClick="cancel"
-        :disablePrimaryButton="readOnly"
-      /> </template
-  ></formio-container>
+      <div v-if="showCompleteInfo">
```

Comment:
Using the `showCompleteInfo` to hide the footer does not seem accurate.
Can we have a property called "showFooter" instead?